docs/book/promotions.rst

```diff
@@ -4,37 +4,173 @@
 Promotions
 ==========
 
-...
+The system of **Promotions** in **Sylius** is really flexible. It is a combination of promotion rules and actions.
+
+Promotions have a few parameters - a unique ``code``, ``name``, ``usageLimit``,
+the period of time when it works.
+There is a possibility to define **exclusive promotions** (no other can be applied if an exclusive promotion was applied)
+and **priority** that is useful for them, because the exclusive promotion should get the top priority.
+
+How to create a Promotion programmatically?
+-------------------------------------------
+
+Just as usually, use a factory. The promotion needs a ``code`` and a ``name``.
+
+.. code-block:: php
+
+   /** @var PromotionInterface $promotion */
+   $promotion = $this->container->get('sylius.factory.promotion')->createNew();
+
+   $promotion->setCode('simple_promotion_1');
+   $promotion->setName('Simple Promotion');
+
+**Of course an empty promotion would be useless** it is just a base for adding **Rules** and **Actions**.
+Let's see how to make it functional.
 
 Promotion Rules
 ---------------
 
-...
+The promotion **Rules** are methods of checking if a promotion should be applied.
```

Comment:
`The promotion **Rules** restrict in what circumstances a promotion will be applied.`
Checking whether it is, takes place in an appropriate **RuleChecker**.